robot/robot.py

```diff
@@ -39,22 +40,18 @@ def __init__(self, robotd_path: Union[str, Path]=ROBOTD_ADDRESS) -> None:
         ]
 
         self._wait_for_power_board()
+        self.power_board.power_on()
+
+        print('Waiting for start button.')
+        self.power_board.wait_start()
+
+        print("Starting user code.")
 
     def _wait_for_power_board(self):
```

Comment:
Can we rename this method to make sense? It no longer does any waiting, and certainly not for the power board.